news/2018-03-27-the-qat-gazette-march-2018.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: "The QAT Gazette: March 2018"
+date: 2018-03-27 09:00:00 +0000
+---
+## Opening 
+by [JBHyperion](https://osu.ppy.sh/users/4879508)
+
+Welcome back everyone! JBHyperion here with another month's worth of news and developments!
+
+Restructuring of the Quality Assurance Team, more communication and transparency regarding decision-making, and a greater focus on actually ensuring quality in the Qualified section again... Big changes to discuss this month, so read inside to find out all the details!
+
+![](/wiki/shared/news/banners/qat-blog-logo.png)
+
+As some of you may already be aware, it's been another busy month in our fair community, though you've hopefully come to expect that - these articles wouldn't be very exciting if there was nothing interesting to talk about after all! Thankfully, it's also been a month of tremendous growth. With growth comes change, and with changes comes an exciting opportunity for us to tell you all about it! 
+
+One of these changes is that for the first time in months, I won't be presenting this issue alone! Who knows what could happen, so let's get started right away!
+
+## Within the Beatmap Nominators 
+by [Protastic101](https://osu.ppy.sh/users/6712747)
+
+Last month we opened the Gazette with some fun little statistics about the current rate of beatmap qualification and the impressive showing made by our hard-working Beatmap Nominators. I'm pleased to say a month later that our progress continues at an incredible pace, with currently just shy of 120 brand new beatmaps on their way to Ranked status.
```

Comment:
Could add a little sentence here stating that this is only going to increase as more Beatmap Nominators join our ranks... which leads nicely into the next section about new additions.

Reply:
done